### ISSUE TITLE
Could not find Visual Studio. VS2015 or VS2017 (with C++) needs to be installed 

### ISSUE THREAD

**Issue by LOrdEndl** [microsoft/vcpkg#8346]
please help for this error

**Comment by LilyWangL** [comment]
Hi @LOrdEndl 
Have you installed Visual Studio on your device?

**Comment by wesleygriffin** [comment]
I'm having this same problem. I used to have VS 2017 installed but recently uninstalled it as I've moved completely to VS 2019. The README states "Visual Studio 2015 Update 3 or newer". Is VS 2019 not supported yet? Here's the output from a freshly cloned vcpkg repo:

~~~
C:\Users\WesleyGriffin\Active\vcpkg>.\bootstrap-vcpkg.bat
The property 'instance' cannot be found on this object. Verify that the property exists.
At C:\Users\WesleyGriffin\Active\vcpkg\scripts\bootstrap.ps1:78 char:31
+         foreach ($instance in $asXml.instances.instance)
+                               ~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], PropertyNotFoundException
    + FullyQualifiedErrorId : PropertyNotFoundStrict

Could not find Visual Studio. VS2015 or VS2017 (with C++) needs to be installed.
At C:\Users\WesleyGriffin\Active\vcpkg\scripts\bootstrap.ps1:143 char:9
+         throw "Could not find Visual Studio. VS2015 or VS2017 (with C ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Could not find ...o be installed.:String) [], RuntimeException
    + FullyQualifiedErrorId : Could not find Visual Studio. VS2015 or VS2017 (with C++) needs to be installed.
~~~

**Comment by wesleygriffin** [comment]
I think I tried to bootstrap while Visual Studio Installer was updating. What threw me off was the error message only mentions 2015 and 2017 and not 2019.

**Comment by LilyWangL** [comment]
Thanks for posting this issue. Closed as duplicate of #7903 and #2075. Please pay attention to the progress of issue #7903 and #2075.
